src/sentry/runner/commands/run.py

```diff
@@ -427,20 +427,35 @@ def post_process_forwarder(**options):
     type=click.Choice(["earliest", "latest"]),
     help="Force subscriptions to start from a particular offset",
 )
+@click.option(
+    "--use-arroyo-consumer",
+    default=False,
+    is_flag=True,
+    help="Switches to the new arroyo consumer implementation.",
+)
```

Comment:
Arroyo consumers handle auto offset reset a bit differently, mind adding the `@strict_offset_reset_option()` decorator here and passing it's value through to the `KafkaConsumer`